qa/integration-tests/src/test/java/io/camunda/it/operate/OperateMultiTenancyIT.java

```diff
@@ -76,6 +77,7 @@ public void beforeEach() {
   }
 
   @Test
+  @Disabled
```

Comment:
:thought_balloon: Let's make sure we at least have a note in an issue somewhere to re-enable it. I don't think I would remember :sweat_smile:

Reply:
Raised an issue https://github.com/camunda/camunda/issues/27160 and linked it to the epic :)